components/frontpage/HeroSection.tsx

```diff
@@ -101,11 +101,11 @@ const HeroSection = () => {
         </Subtitle>
 
         <ButtonsWrapper>
-          <Link href="/contribute">
+          <Link href="/contribute" passHref>
```

Comment:
is passHref needed?
As per the docs: "passHref - Forces Link to send the href property to its child". I don't see why "button" should have a href?

Reply:
This wasn't necessary to build but there were dozens of warnings in the Vercel build logs and I felt it worthwhile to give it what it wanted. Warnings do not cause a build failure.

Reply:
Hmm, I see. Did it say why it gave this warning?
Looking at the docs, I don't see why we would need this. It also seems really redundant to do it to _all_ links.

Of course, if it complains it should be fixed - I'd just like to know exactly why :)

Reply:
Sure... the build log is below as an image:

![image](https://user-images.githubusercontent.com/75389/134788451-3124d0c2-de1d-4da2-aace-8cdfe0d02b9d.png)

Reply:
Ahh, I see. I read up on a bit more, and it _seems_ that it's only needed if the links wraps an a-tag.
Can you go through them again, and make sure it only applies if they wrap an a-tag :) ?

Reply:
Actually, you can keep this as is if you want. I see it gives some eslint warnings if we remove it, and it seems like it works with a-tags but with weird behavior .. We'll have to investigate this some more some day :)